pkg/cidata/user-data.TEMPLATE

```diff
@@ -99,7 +236,7 @@ write_files:
       fi
       {{- end}}
    owner: root:root
-   path: /var/lib/cloud/scripts/per-boot/10-install-packages.boot.sh
+   path: /var/lib/cloud/scripts/per-boot/30-install-packages.boot.sh
    permissions: '0755'
 {{- end}}
 {{- if or .Containerd.System .Containerd.User}}
```

Comment:
`.Containerd.System` and `.Containerd.User` should be ignored with a warning message on non-systemd hosts, until we can have openrc scripts

Reply:
Yes, but the "warning message" will only be in `/var/log/cloud-init-output.log` on the guest. We don't know if the OS is using systemd or openrc at the time the template is rendered, and I intentionally didn't want to add another config setting for it.

I guess I could add a "requirement" for it though, to check once the VM is up and running.

Reply:
I've tried to implement this in d981c4f but am not really happy with it:

1. The `fatal` field seems like a kludge, but there is no point in retrying the check for `systemd`; it isn't going to magically appear eventually.
2. The failure output is really ugly/hard to read. I suspect all requirements failures look like this now though, so maybe not something to address in this PR?

```
INFO[0035] [hostagent] Waiting for the optional requirement 1 of 2: "systemd must be available"
INFO[0035] [hostagent] Forwarding "/run/user/501/lima-guestagent.sock" (guest) to "/Users/jan/.lima/lima/ga.sock" (host)
INFO[0035] [hostagent] No further optional requirements will be checked
ERRO[0035] [1 error occurred:
	* failed to satisfy the optional requirement 1 of 2 "systemd must be available": systemd is required to run containerd, but does not seem to be available.
Make sure that you use an image that supports systemd. If you do not want to run
containerd, please make sure that both 'container.system' and 'containerd.user'
are set to 'false' in the config file.
; skipping further checks: stdout="", stderr="+ command -v systemctl\n+ echo 'systemd is not available on this OS'\nsystemd is not available on this OS\n+ exit 1\n": failed to execute script "systemd must be available": stdout="", stderr="+ command -v systemctl\n+ echo 'systemd is not available on this OS'\nsystemd is not available on this OS\n+ exit 1\n": exit status 1

]
WARN[0035] DEGRADED. The VM seems running, but file sharing and port forwarding may not work. (hint: see "/Users/jan/.lima/lima/ha.stderr.log")
FATA[0035] degraded, status={Running:true Degraded:true Exiting:false Errors:[1 error occurred:
	* failed to satisfy the optional requirement 1 of 2 "systemd must be available": systemd is required to run containerd, but does not seem to be available.
Make sure that you use an image that supports systemd. If you do not want to run
containerd, please make sure that both 'container.system' and 'containerd.user'
are set to 'false' in the config file.
; skipping further checks: stdout="", stderr="+ command -v systemctl\n+ echo 'systemd is not available on this OS'\nsystemd is not available on this OS\n+ exit 1\n": failed to execute script "systemd must be available": stdout="", stderr="+ command -v systemctl\n+ echo 'systemd is not available on this OS'\nsystemd is not available on this OS\n+ exit 1\n": exit status 1

] SSHLocalPort:60022}
```

Suggestions welcome!